rules/style.js

```diff
@@ -121,7 +121,11 @@ module.exports = {
     "operator-linebreak": [2, "after"],
     // Enforce padding within blocks
     // http://eslint.org/docs/rules/padded-blocks
-    "padded-blocks": [2, "never"],
+    "padded-blocks": [2, {
+      "blocks": "never",
+      "classes": "always",
```

Comment:
i don't think classes were a part of this rule before. new options were added recently.

as for the more human-friendly `error` and `warning`, i didn't think it warranted a change. but i'm open to doing a find & replace.